app/e2e/common/global/test-setup.ts

```diff
@@ -0,0 +1,24 @@
+import { chromium } from '@playwright/test';
+import { PlaywrightLandingPage } from '@local/common/pages/playwright-landing-page';
+
+async function globalSetup() {
+    // Setup Context
+    const browser = await chromium.launch();
+    const context = await browser.newContext({ baseURL: 'http://localhost:8080' });
+    // Open new page
+    const page = await context.newPage();
+    const landingPage = new PlaywrightLandingPage(page);
+    await landingPage.goto();
+    // Create Account
+    await landingPage.appBarRegister({
+        email: `test@example.com`,
+        password: 'Password1!',
+        firstName: 'First',
+        lastName: 'Last',
+    });
+    // Save signed-in state to 'storageState.json'
+    await page.context().storageState({ path: 'storageState.json' });
+    await browser.close();
```

Comment:
If, for some reason, creating accounts on the register page was broken, all tests would be broken, which is part of the reason I'm not a fan of this.

Reply:
I don't think that's an issue, if creating accounts is broken then there are some big problems that need to be fixed before the PR causing them to break would ever be merged.